lib/rgeo/geos/interface.rb

```diff
@@ -29,30 +29,30 @@ def supported?
       # Returns true if the given feature is a CAPI GEOS feature, or if
       # the given factory is a CAPI GEOS factory.
 
-      def is_capi_geos?(object_)
+      def is_capi_geos?(obj)
```

Comment:
`object`. (it's not a keyword, so it's ok as a variable name).  The comments in this file were changed too.